lib/tasks/import_staticstats.rake

```diff
@@ -0,0 +1,93 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :staticstats, [:csv_file] => [:environment] do |t, args|
+
+    habitats = Habitat.all
+    
+    habitats.each do |habitat|
+      csv_file = "#{habitat.name.capitalize}_Protected.csv"
+      Rails.logger.info "import CSV: #{csv_file}"
+      import_csv_file(habitat.name, csv_file)
+      
+      csv_file = "#{habitat.name.capitalize}_Total.csv"
+      Rails.logger.info "import CSV: #{csv_file}"
+      import_csv_file(habitat.name, csv_file)     
+    end
+
+    puts "recalculate protected percentages"
+    recalculate_protected_percentages
+
+  end
+  
+  def import_csv_file(habitat, csv_file)
+    filename = "#{Rails.root}/lib/data/#{csv_file}"
+    
+    csv = File.open(filename, encoding: "utf-8")
+    csv_headers = File.readlines(csv).first.split(",")
+    
+    staticstats_hash = {
+      iso3: csv_headers[0],
+      value: csv_headers[1].chomp
+    }
+    
+    CSV.parse(csv, headers: true, encoding: "utf-8") do |row|
+      csv_staticstats_row = row.to_hash
+      staticstats_row_hash = {}
```

Comment:
Also this hash seems not to being used anywhere

Reply:
Good point! Thanks!